packages/lib/src/doc-utils/upload-handler.ts

```
@@ -0,0 +1,130 @@
+	duration?: number
+	className?: string
+}
+
+export const useHandleFileUpload = (
+	documentType: string | null,
+	setFile: (file: File | null) => void,
+	setPreviewUrl: (url: string | null) => void,
+	setIsProcessing: (isProcessing: boolean) => void,
+	setProgress: (progress: number) => void,
+	setValidationErrors: (errors: string[]) => void,
+	setExtractedData: (data: any) => void,
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)